src/main/java/com/adobe/target/edge/client/http/TargetHttpClient.java

```diff
@@ -13,14 +13,13 @@
 
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
-import kong.unirest.HttpResponse;
 
 public interface TargetHttpClient extends AutoCloseable {
 
-  <T, R> HttpResponse<R> execute(
+  <T, R> ResponseWrapper execute(
```

Comment:
Let's use `ResponseWrapper<R>` for sake of consistency

Reply:
fixed